test/functional/feature_rbf.py

```diff
@@ -545,24 +542,25 @@ def test_prioritised_transactions(self):
         assert tx2b_txid in self.nodes[0].getrawmempool()
 
     def test_rpc(self):
-        us0 = self.nodes[0].listunspent()[0]
+        us0 = self.wallet.get_utxo()
         ins = [us0]
-        outs = {self.nodes[0].getnewaddress(): Decimal(1.0000000)}
+        outs = {"bcrt1qxcm2aqvzejqyd44jujp5qyz6sya035j6st07g0": Decimal(1.0000000)}
```

Comment:
can this use `ADDRESS_BCRT1_UNSPENDABLE`?

Reply:
Yes, I think it can (I have succesfully run the test locally), but would there be any benifit of using it in place of the hardcode address present currently?

Reply:
Obviously a nit, but using the undependable address clarifies that the test isn't going to spend this output

Reply:
Agreed, thanks for clarifying. I have included it now.